config/config.yml

```diff
@@ -15,6 +15,8 @@ SESSION_SECRET: '898f6d0363863ec79d782238cd1c5767636d712cc0d138238bcd5bfc9d2672f
 RETURN_ADDRESS: 'do-not-reply@example.org'
 SUPPORT_ADDRESS: 'support@example.org'
 ABUSE_ADDRESS: 'abuse@example.org'
+SPAM_ALERT_ADDRESS: 'otw-abuse-discuss@transformativeworks.org'
```

Comment:
All of the other addresses in this file are example addresses, not real ones, presumably to keep Abuse and Support from getting spammed with stuff from test/dev/etc. Should this be an example address, too?